### PR TITLE
Some more cleanups.

### PR DESCRIPTION
**Pull request by yossigo**
* Redis Cluster compatible errors codes (`-CLUSTERDOWN` instead of `-NOLEADER`).
* Configurable source repositories.

### CODE REVIEW

**Comment by yossigo** [comment]
Hi @aphyr, pinging you about another couple of small changes to merge. Thanks!

File: src/jepsen/redis/core.clj [hunk 1]
```diff
@@ -139,7 +139,10 @@
     :default true]
 
    [nil "--raft-version VERSION" "What version of redis-raft should we test?"
-    :default "e0123a9"]
+    :default "master"]
```
Comment (aphyr) [inline src/jepsen/redis/core.clj]: Do you want this to be non-reproducible?

File: src/jepsen/redis/core.clj [hunk 2]
```diff
@@ -161,8 +164,11 @@
    [nil "--tcpdump" "Create tcpdump logs for debugging client-server interaction"
     :default false]
 
-   ["-v" "--version VERSION" "What version of Redis should we test?"
-    :default "6.0.3"]
+   [nil "--redis-version VERSION" "What version of Redis should we test?"
+    :default "unstable"]
```
Comment (aphyr) [inline src/jepsen/redis/core.clj]: Ditto here--up to y'all, but this might make it difficult for you to do regression testing

Comment (yossigo) [reply src/jepsen/redis/core.clj]: Yes, I considered that but having a fixed stale version is also not the best. I'll just drop the default to make everything explicit.

Comment (aphyr) [reply src/jepsen/redis/core.clj]: That's fine! Or you can do unstable here! Either way, just wanted to check. :-)

**Comment by aphyr** [comment]
Righto!

**Pull request merged**